packages/atst/docs/cli.md

```diff
@@ -1,5 +1,7 @@
 # atst cli docs
 
+![preview](https://user-images.githubusercontent.com/35039927/222435290-0271bc85-1e62-4d0a-b539-084af1e22ded.gif)
```

Comment:
Probably best to do a relative link here
```suggestion
![preview](../assets/preview.gif)
```

Reply:
Oh good idea then we update these automatically when the gif changes

Reply:
```suggestion
![preview](../assets/preview.gif)
```
Force pushing accidentally wiped these changes